App.tsx

```diff
@@ -135,7 +136,7 @@ const App = () => {
   
   return (
     <NavigationContainer theme={navigationTheme} linking={linking}>
-    <Stack.Navigator initialRouteName={initialPage} screenOptions={{ headerBackTitleVisible: false }}>
+    <Stack.Navigator initialRouteName={initialPage} screenOptions={{ headerBackTitleVisible: false,  headerTintColor: 'white', headerStyle: { backgroundColor: '#3377FF' } }}>
```

Comment:
We should not have any colors by hex number in the pages, we need to bring them in from the themes